src/main/resources/findologic_20.xsd

```diff
@@ -305,7 +338,7 @@
 
     <xs:complexType name="overriddenPricesType">
         <xs:sequence>
-            <xs:element name="overriddenPrice" minOccurs="0" maxOccurs="unbounded">
+            <xs:element name="overriddenPrice" minOccurs="1" maxOccurs="unbounded">
```

Comment:
Check https://github.com/findologic/xml-export/pull/15/files#r995640691
Maybe we can simplify/merge some types here

Reply:
I've simplified this a bit and created a new type `basePriceType` that is used by price and overriddenPrice